router/router.go

```diff
@@ -233,8 +233,14 @@ func New(cfg *config.Configuration, rateConvertor *currencies.RateConverter) (r
 		glog.Fatalf("Failed to create the amp endpoint handler. %v", err)
 	}
 
+	videoEndpoint, err := openrtb2.NewVideoEndpoint(theExchange, paramsValidator, fetcher, cfg, r.MetricsEngine, pbsAnalytics, disabledBidders, defReqJSON, bidderMap, categoriesFetcher)
+	if err != nil {
+		glog.Fatalf("Failed to create the simplified openrtb endpoint handler. %v", err)
```

Comment:
Identical error message as for an earlier call.  Please make this specific to video so they can be distinguished.

Reply:
Fixed